app/backend/app.py

```diff
@@ -308,9 +308,10 @@ async def setup_clients():
         else:
             endpoint = f"https://{AZURE_OPENAI_SERVICE}.openai.azure.com"
 
-        # Store on app.config for later use inside requests
+        api_version = os.environ.get("AZURE_OPENAI_API_VERSION", "2024-03-01-preview")
```

Comment:
```suggestion
        api_version = os.getenv("AZURE_OPENAI_API_VERSION", "2024-03-01-preview")
```

slightly shorter wrapper func